src/ProtoRoute/Ghcid.hs

```diff
@@ -12,7 +12,8 @@ runGhci stuff = do
     curDir <- getCurrentDirectory
     (g, _) <- startGhci "ghci" (Just curDir) f
     let runCmd s = execStream g s f
+    runCmd ":set -XOverloadedStrings"
     runCmd ":load src/ProtoExports"
-    runCmd "import ProtoRoute.Message"
     runCmd ("let testMsg = " ++ stuff)
+    runCmd "testMsg"
```

Comment:
It's better to print serialized representation of `testMsg` since our goal eventually is to get binary serialized representation of data type.